tests/PHPStan/Analyser/data/in-array-non-empty.php

```diff
@@ -14,15 +14,15 @@ class HelloWorld
 	public function sayHello(array $array): void
 	{
 		if(in_array("thing", $array, true)){
-			assertType('non-empty-array<int, string>', $array);
+			assertType('non-empty-array<int, string>&hasOffset(mixed)', $array);
```

Comment:
this is obviously weird. I'd prefer to rather have an `hasOffsetValue('thing')` again I guess. But on the other hand, the `HasOffset` with both the offset and offset value might be useful somewhere else?

Reply:
maybe print offset and offsetValue type within the bracket, like
`&hasOffset(mixed, 'thing')` ?

Reply:
that would be easy to do and makes sense, yeah. I did not do it yet because I was scared that it will lead to too many changes in e.g. baselines. but we can't think always about that, right? I'll check it out :)